include/spirv-tools/libspirv.h

```diff
@@ -24,6 +24,8 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 
+#include <spirv/unified1/spirv.h>
```

Comment:
This part makes me sad.  It's suboptimal to add a new project / include-dir dependency for this one feature.